src/gui_common/CustomPopupMenu.cs

```diff
@@ -13,8 +13,8 @@ public class CustomPopupMenu : TopLevelContainer
     public NodePath ContainerPath = null!;
 
 #pragma warning disable CA2213 // Disposable fields should be disposed
+    protected Container container = null!;
```

Comment:
Why are these fields reordered? They no longer match the order of the NodePaths so I think this is a change that makes this worse...

Reply:
I missed that access modifier change. Why is the container needed there? Why not add the code that handles the container sizing directly to this class? For example as a protected method if there's no other way to architect the flow of the required size calculation.

Reply:
The reason is that the title text which is a child of the container only exists in the OrganellePopupMenu. Though, considering this now, I probably should've just used an exported node path in a derivative class to get that instead of switching this to protected. Will fix this.

Reply:
Protected method works, tho I don't know what to necessarily call it due to the lack of understanding of what the code does. Best guess currently to name it is ForceSizeUpdate or ForceContainerRedraw since those are the two things names that make sense in current guesses of what this is doing. (I'm partial to the latter given it is more sure that it is at least doing that at this time compared to the former which is maybe a side effect of the redraw which may be what is used.)

Reply:
Maybe something like `WorkaroundContainerSizeProblem`? That'd be very explicit that the method exists just to get around a problem that should not exist in an ideal world.